model/src/main/kotlin/VcsInfo.kt

```diff
@@ -83,6 +83,11 @@ data class VcsInfo(
      * Return a [VcsInfoCurationData] with the properties from this [VcsInfo].
      */
     fun toCuration() = VcsInfoCurationData(type, url, revision, path)
+
+    /**
+     * Return true if this vcs information matches the other vcs information.
```

Comment:
s/vcs/VCS/

Reply:
The docs for this function require some special attention: The matching disregards the `path` property (otherwise just `vcsInfoA == vcsInfoB` could have been used for matching). So in a way this is a bit similar to a `PackageCuration`'s `isApplicableDisregardingVersion()` function, and I'd name it as `equalsDisregardingPath()`.

Reply:
When I change the name and the doc, should the same doc changes be applied to the `matches` method in `RespositoryProvenance` in the next commit:
https://github.com/oss-review-toolkit/ort/pull/8764/commits/f43b5d9c97c427d8b8ae41561514660e44828116#diff-db17c034876de657175ab28d452f2812913baba034a764c609990260ce101751

Reply:
Not necessarily. The current docs already make clear the only the normalized VCS info is compared, so I think we're good.

Reply:
Done.